kombu/transport/pubsub.py

```diff
@@ -212,10 +221,22 @@ def _msg_get(self, subscription_path):
             for msg in resp.received_messages:
                 if self.temp_cache[subscription_path].full():
                     break
+                data = loads(msg.message.data)
```

Comment:
this can still fail in case data is not stringified json?

Reply:
@jeechu yes, it will .. i thought of it but expected that message which will be published will be a stringified json. Will update it if that's not the case .

Reply:
please update since we want to avoid the process halting

Reply:
changes done